processors/base/writers/location.py

```diff
@@ -34,7 +40,7 @@ def write_location(conn, location, source_id, trial_id=None):
     timestamp = datetime.datetime.utcnow()
 
     # Get name
-    name = helpers.clean_string(location['name'])
+    name = helpers.clean_string(extract_CanonicalName(location['name']))
```

Comment:
I see. Still, if you think any match with >80% similarity is good, you could break the code after the first one is found (avoiding having to check all countries every time). Still, I'd guess returning the closest one (and not the last with >80% similarity) can give better results, even though I'd agree it should be quite rare to have close ties.

Are you sure that `fuzz.ratio()` lowercases/normalizes the names? I tried looking into their code, but couldn't find this normalization. Maybe I'm missing something, but it's better to confirm.